data/human/culture independents.txt

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+# Courtesy of ES Celestia
+mission "ZZCU Tarazed Ship Christening"
+	minor
+	source "Wayfarer"
+	to offer
+		random == 1
+		has "main plot complete"
+	on offer
+		conversation
+			`As you walk past a number of ships and crew going about their work in the spaceport, a small gathering in front of a Shuttle catches your attention.  A young woman wearing a rather cheesy pilot uniform is in the midst of a little group of people all celebrating together.  As you pass by them, a man standing on the top of a ladder breaks a bottle of Champagne on the (clearly not new) ship's hull.  A loud cheer erupts from the group, earning an embarrassed blush from the young woman.`
+			choice
+				`	(Take a closer look.)`
+					goto investigate
+				`	(Shout well wishes as you pass by)`
+					goto passby
+				`	(Ignore them, you have more important things to do.)`
+			`	You hurry past the rowdy group, annoyed by the disturbance.`
+				decline
+			label passby
+			`	You hurry past the group as they quiet down.  Taking advantage of the momentary lull, you shout "Good luck!  You've got this!" towards them.  They let out another cheer in response, with the exception of the young woman who buries her face in her hands.`
+				decline
+			label investigate
+			`	You approach the group as they quiet down, noticing hint of anxiety in the air.`
+			`	"Nothing quite like buying your very first starship," you say over their chatter.  The celebration goes quiet as they turn to see who interrupted their little party.`
+			`	"It's certainly been an...experience."  The young woman replies once you are in comfortable talking distance.`
+			choice
+				`	"Have you got any big plans with it?"`
+					goto plans
+				`	(Give some advice.)`
+					goto advice
+				`	(Make a bad joke.)`
```

Comment:
I try to avoid mixing "action" and "dialog" choices, especially when the action choices are just to say some dialog. You should take the dialog that follows these two action choices and just have them be the choices themselves.